src/main/kotlin/uk/co/poolefoundries/baldinggate/core/CoreComponents.kt

```diff
@@ -98,6 +112,13 @@ data class TextureRenderable(val texture: Texture) : Renderable {
     }
 }
 
+data class MobRenderable(val entity: Entity, val renderable: Renderable) : Renderable {
```

Comment:
Let's enhance this to draw a HP and AP bar below the mob

Reply:
Love this idea.